themes/default/views/editor/objects/screen_html.php

```diff
@@ -82,4 +82,56 @@
 
 	<div class="editorBottomPadding"><!-- empty --></div>
 	
-	<?php print caSetupEditorScreenOverlays($this->request, $t_object, $va_bundle_list); ?>
\ No newline at end of file
+	<?php print caSetupEditorScreenOverlays($this->request, $t_object, $va_bundle_list); ?>
+	<script type="text/javascript">
```

Comment:
My first reaction was, "oooh look at mister javascript over here" ;-)  I do like it -- the code pattern you've used here is valid (e.g. UMD), but it's overkill for this situation.  However, that's something we can discuss further just in a more general sense, because...

In any case, and as much as it kills me to "enforce" this: we should stick to the CA convention of using `jQuery` directly (without specifying `window` which is implied) everywhere, and not defining a `$` variable as a shortcut (or using the `$` that jQuery also puts into the global scope.  This removes the need for the wrapping iiaf structure, i.e. you could remove lines 87-92 and lines 134-135 (but changing `$` to `jQuery` throughout).